### DISCUSSION TITLE
Is a new Snapcraft built in the works?

### DISCUSSION THREAD

**Discussion by ddppddpp** [semaphoreui/semaphore#2078] [General]
Is there a reason why the current snap built is 3 months older than the current release? Is Snapcraft still considered as a prime chocie for installations?

**Comment by tboerger** [comment]
No official decision since I'm a contributor only. But after having various issues related to snaps I have suggested to deprecate snaps.

**Reply by ddppddpp** [reply]
Sounds reasonable. Maybe the docs should be updated or at least a discussion topic should be stickied/F.A.Q.ed here or whatever.

**Comment by fiftin** [comment]
Hi @ddppddpp 
The Snap variant is not as popular as the Docker variant, but it is difficult to support and test.
